jobs/delorean/jenkinsfiles/2.0/Jenkinsfile

```diff
@@ -0,0 +1,29 @@
+pipeline {
+    agent none
+
+    parameters {
+
+        // https://datagrepper.engineering.redhat.com/id?id=ID:messaging-devops-broker02.web.prod.ext.phx2.redhat.com-44775-1580379079663-8:893046:0:0:1&is_raw=true&size=extra-large
+        string defaultValue: '', description: '', name: 'CI_MESSAGE', trim: true
+    }
+
+    stages {
+        stage('Send mail') {
+            steps {
+                script {
+                    final Map msg = readJSON text: CI_MESSAGE
+                    // Sending this mail is just to test things for now
+                    // TODO Replace with pr creation
+                    mail(
+                        subject: "New build of 3scale created in Brew",
+                        to: 'lfitzger@redhat.com',
+                        body: [
+                            "A brew build was created for 3scale",
+                            "MSG: ${msg.info.package_id}"
+                        ].join('\n')
+                    )
+                }
+            }
+        }
+    }
+}
```

Comment:
This should be in a different directory which is named according to what the jenkinsfile does, umb would do for now i guess jobs/delorean/jenkinsfiles/2.0/umb/Jenkinsfile